doc/Tex/RAM_SCB_install.tex

```diff
@@ -1,180 +1,72 @@
-NetCDF requires only a C compiler.  {\tt gcc} is strongly suggested.
-
-\subsubsection{Pspline Quick Install}
-Pspline requires NetCDF, so be sure that NetCDF is correctly installed before 
-continuing.  After unpacking the tarball, compile the library using
-\begin{verbatim}
-make FORTRAN_VARIANT=Portland NETCDF_DIR=~/netcdf_lib
-\end{verbatim}
-It is important to specify the Fortran compiler that coincides with the 
-compiler you will use to make the RAM-SCB executable.  Other options accepted 
-by the {\tt FORTRAN\_VARIANT} variable are {\tt PathScale},  {\tt GCC}(gfortran
-and g95), {\tt NagWare},  {\tt Fujitsu}, {\tt Intel}, and {\tt Absoft}.  
-Ensure that the value of {\tt NETCDF\_DIR} coincides with the installation 
-directory of NetCDF.
-
-{\tt make} may fail during the creation of the Pspline test utilities.  If 
-this happens, check for the creation of a {\tt lib} directory and several 
-\verb*l*.al files contained within.  If they exist, disregard the error 
-message and continue to the next step.
-
-Install Pspline as follows (changing the value of {\tt PREFIX} to select 
-installation directory):
-\begin{verbatim}
-make install PREFIX=~/psline_portland
-\end{verbatim}
-On machines where you will use different compilers, it is helpful to add the 
-compiler name to the name of the installation directory.
+% QUESTION: All of the information in this section is out of date. Even the NETCDF installation instructions are incorrect given that they have split it into a C version and a Fortran version. This is still easy enough to describe and walk through, but the installation of GSL from source isn't as easy. Do we really want to have this section?
 
 \subsection{Libraries From Modules}
-Well maintained clusters often use the {\tt module} interface for loading
-and unloading software packages.  Be sure to explore the available packages
-on a new system before going through the arduous task of installing from source!
+Well maintained clusters often use the {\tt module} interface for loading and unloading software packages.  Be sure to explore the available packages on a new system before going through the arduous task of installing from source!
 
 \begin{verbatim}
 module avail
 \end{verbatim}
 \noindent
-will list all available software libraries on the system.  You can load a 
-library, unload a library, and list loaded software using the module commands:
+will list all available software libraries on the system.  You can load a library, unload a library, and list loaded software using the module commands:
 
 \begin{verbatim}
 module load package_name
 module unload package_name
 module list
 \end{verbatim}
 
-As an example, here is what you would type on NASA's Pleiades cluster to get
-started with RAM-SCB:
+As an example, here is what you would type on NASA's Pleiades cluster to get started with RAM-SCB:
+% QUESTION: ^Is this even still true? Seems silly to have to keep up with changes on Pleiades in our documentation
```

Comment:
No idea. I haven't used Pleiades and I don't know what the range of modules there are. Having an example is useful though, so maybe we just switch to the grizzly modules and say it's an illustrative example. Probably also good to mention `module purge` to ensure that everything is unloaded...